login-workflow/src/screens/BLCSelfRegistrationSuccessScreen/BLCSelfRegistrationSuccessScreen.tsx

```diff
@@ -0,0 +1,119 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import Typography from '@mui/material/Typography';
+import { SuccessScreenBase, SuccessScreenProps } from '..';
+import { useRegistrationWorkflowContext, useRegistrationContext } from '../../contexts';
+import Box from '@mui/material/Box';
+import { AccountCircle, Business } from '@mui/icons-material';
+
+/**
+ * Component that renders a success screen for when registration completes.
+ *
+ * @param icon the icon to be displayed on the screen
+ * @param messageTitle title of the success message
+ * @param message success message to be displayed on the screen
+ * @param onDismiss function to call when user clicks button
+ * @param canDismiss function to call when the dismiss button is clicked
+ *
+ * @category Component
+ */
+
+const BLCSelfRegistrationSuccessIcons = (): JSX.Element => (
+    <Box sx={{ display: 'flex' }}>
+        <Box
+            sx={{
+                display: 'flex',
+                justifyContent: 'center',
+                alignItems: 'center',
+                width: '100px',
+                height: '100px',
+                backgroundColor: '#e0eff8',
+                borderRadius: '50%',
+            }}
+        >
+            <AccountCircle color={'primary'} sx={{ fontSize: 54 }} />
```

Comment:
<AccountCircle/> and <Business/> icons are not going to be changed in the future right? Since we have hardcoding this.